### PR TITLE
Release Process Updates

### PR DESCRIPTION
**Pull request by caryr35**
This PR is add  item 'Final call for new release artifacts in PIC synch' to the Release Burndown process

### CODE REVIEW

File: releases/process.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Burn down is the process of locking down all issues slated for the release and m
 
 ### Timing
 
-For the selection of a `burn down date`, the general guidelines should be followed:
+For the selection of a `burn down date`, the genereral guidelines should be followed:
```
Comment (raydouglass) [inline releases/process.md:58]:
```suggestion
For the selection of a `burn down date`, the general guidelines should be followed:
```

File: releases/process.md [hunk 2]
```diff
@@ -105,10 +105,11 @@ Generally the process for Code Freeze occurs around 10:00AM PT on the first day
 Project Leads
 {: .label .label-blue}
 
-1. Move any open pull requests targeting `branch-M.B` to target `branch-M.C` instead
-2. Wait for confirmation from operations on the branch switch
-3. Continue `M.C` development
-4. Respond promptly to operations if any issues are found with the `M.B` release
+1. Final call for new release artifacts in PIC synch (2 weeks prior to cuDF burndown)
```
Comment (aravenel) [inline releases/process.md:108]:
```suggestion
1. Inform operations team of any new release artifacts (packages, wheels, containers) no later than 2 weeks prior to cuDF burndown.
```

Comment (aravenel) [reply releases/process.md:108]: Didnt want to mention PIC sync as these docs are public and PIC syncs are internal NV-only.

**Review by aravenel — CHANGES_REQUESTED** [review]

File: releases/process.md [hunk 3]
```diff
@@ -105,10 +105,11 @@ Generally the process for Code Freeze occurs around 10:00AM PT on the first day
 Project Leads
 {: .label .label-blue}
 
-1. Move any open pull requests targeting `branch-M.B` to target `branch-M.C` instead
-2. Wait for confirmation from operations on the branch switch
-3. Continue `M.C` development
-4. Respond promptly to operations if any issues are found with the `M.B` release
+1. Inform operations team of any new release artifacts (packages, wheels, containers) no later than 2 weeks prior to cuDF burndown.
```
Comment (bdice) [inline releases/process.md:108]: Small edit: remove trailing period to align with the other items in this list.
```suggestion
1. Inform operations team of any new release artifacts (packages, wheels, containers) no later than 2 weeks prior to cuDF burndown
```

**Review by bdice — APPROVED** [review]
This looks fine to me. @aravenel Can you review this? You requested changes so I want to be sure you're on board before merging. (Just doing a bit of cleanup of PRs in this repo so I came across it again)

**Review by aravenel — APPROVED** [review]
LGTM. Thanks @bdice!

**Pull request merged**